### ISSUE TITLE
hadoop importer parses period wrong if TZ!=utc

### ISSUE THREAD

**Issue by sixtus** [druid-io/druid#228]
This shouldn't happen, but

"intervals": ["2013-08-19T00:00:00Z/2013-08-20T00:00:00Z"]

results in

index-generator-[2013-08-19T00:00:00.000+02:00/2013-08-20T00:00:00.000+02:00]

i.e. "Z" is interpreted as local time zone, and it sure shouldn't be.

**Comment by leventov** [comment]
Closing this after #4611. If the problem persists let's open another issue
